package.json

```diff
@@ -72,6 +72,7 @@
     "typescript": "^3.8.3"
   },
   "resolutions": {
+    "ethers": "5.0.5",
```

Comment:
I wonder if we should instead pin this in resolve-kit.  I worry that it's hard to track all these resolutions, why we did them, and when to remove them.  It might be easier to do that (in this case) if it was part of resolve-kit.  Thoughts?

Reply:
It seems a resolution here is the only way to make it work because an indirect dependency pulls it too:

Before this change:
```
❯ yarn why ethers
=> Found "ethers@5.5.4"
info Reasons this module exists
   - "_project_#@celo#mobile#@valora#resolve-kit" depends on it
   - Hoisted from "_project_#@celo#mobile#@valora#resolve-kit#ethers"
   - Hoisted from "_project_#@celo#mobile#@valora#resolve-kit#@ensdomains#ensjs#ethers"
```

Reply:
@silasbw @jeanregisser is this ok to merge?

Reply:
YES :partying_face: